app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala

```diff
@@ -25,75 +26,123 @@ import scala.util.matching.Regex
 @Singleton
 class CompaniesHouseStubController extends InjectedController {
 
-  private val companyNameKey = "company_name"
-  private val companyNumberKey = "company_number"
-  private val dateOfIncorporationKey = "date_of_creation"
+  private val CharitableIncorporatedOrganisation: Regex = "CE(.*)".r
   private val stubCompanyName = "Test Company Ltd"
-  private val stubCharityName = "Test Charity"
-  private val stubDateOfIncorporation = "2020-01-01"
   private val stubOldDateOfIncorporation = "2017-01-01"
-  private val registeredOfficeAddressKey = "registered_office_address"
-  private val stubRegisteredOfficeAddress = Json.obj(
-    "address_line_1" -> "testLine1",
-    "address_line_2" -> "test town",
-    "care_of" -> "test name",
-    "country" -> "United Kingdom",
-    "locality" -> "test city",
-    "po_box" -> "123",
-    "postal_code" -> "AA11AA",
-    "premises" -> "1",
-    "region" -> "test region"
+
+  private val ukRegisteredOfficeAddress = RegisteredOfficeAddress(
+    address_line_1 = "testLine1",
+    address_line_2 = "test town",
+    care_of = "test name",
+    country = "United Kingdom",
+    locality = "test city",
+    po_box = "123",
+    postal_code = Some("AA11AA"),
+    premises = "1",
+    region = "test region"
   )
 
-  private val stubRegisteredOfficeAddressOverseas = Json.obj(
-    "address_line_1" -> "Overseas Line 1",
-    "address_line_2" -> "Overseas Line 2",
-    "care_of" -> "test name",
-    "country" -> "Ireland",
-    "locality" -> "Overseas City",
-    "po_box" -> "123",
-    "premises" -> "1",
-    "region" -> "Overseas Region"
+  private val overseasRegisteredOfficeAddress = RegisteredOfficeAddress(
+    address_line_1 = "Overseas Line 1",
+    address_line_2 = "Overseas Line 2",
+    care_of = "test name",
+    country = "Ireland",
+    locality = "Overseas City",
+    po_box = "123",
+    postal_code = None,
+    premises = "1",
+    region = "Overseas Region"
   )
 
-  val CharitableIncorporatedOrganisation: Regex = "CE(.*)".r
+  private val defaultUkRegisteredOfficeAddress = RegisteredOfficeAddress(
+    address_line_1 = "10 HIGH STREET",
+    address_line_2 = "address line 2",
+    care_of = "test name",
+    country = "United Kingdom",
+    locality = "TELFORD",
+    po_box = "123",
+    postal_code = Some("TF3 4ER"),
+    premises = "1",
+    region = "SHROPSIRE"
```

Comment:
Missing a 'H' => Shropshire. Also, looking into the call we make it doesn't look like care_of, po_box and region are returned to us so maybe remove them incase Test and Release (who requested the data) complains. Maybe put Telford in address line 2 and then Shropshire can be region

Reply:
ok